roles/sap_hana_preconfigure/tasks/SLES/configuration.yml

```diff
@@ -76,3 +67,7 @@
 - name: Ensure solution was successful
   ansible.builtin.command: "saptune solution verify {{ sap_hana_preconfigure_saptune_solution }}"
   changed_when: no # We're only checking, not changing!
+
+- name: Ensure sapconf and tuned are stopped
+  ansible.builtin.command: "saptune service takeover"
+  changed_when: no # We're only checking, not changing!
```

Comment:
Done

Reply:
@rainerleber Do we need `saptune service enablestart` for completeness? Otherwise I am happy to approve the fix, I just want to be sure the services remain disabled and saptune starts at boot.

Reply:
It is already implemented here: https://github.com/sap-linuxlab/community.sap_install/blob/f0deb287aaa0fb7d992b95a04b14e263b6690c23/roles/sap_hana_preconfigure/tasks/SLES/configuration.yml#L17

It's a different way but I would avoid using shell modules where ever possible

Reply:
I'm good with that explanation, just being devil's advocate following the SLES guide. Approved 👍

Reply:
that is why i appreciate your opinion so much 😊